lib/experimental/new_embed.dart

```diff
@@ -645,6 +647,111 @@ class FlashBox {
   void hide() {
     _element.setAttr('hidden');
   }
+
+  void show() {
+    _element.clearAttr('hidden');
+  }
+}
+
+class AnalysisResultsController {
+  static const String _noIssuesMsg = 'no issues';
+  static const String _hideMsg = 'hide';
+  static const String _showMsg = 'show';
+
+  static const Map<String, List<String>> _classesForType = {
+    'info': ['issuelabel', 'info'],
+    'warning': ['issuelabel', 'warning'],
+    'error': ['issuelabel', 'error'],
+  };
+
+  DElement flash;
+  DElement message;
+  DElement toggle;
+  bool _hidden = true;
```

Comment:
There are a lot of things being shown and hidden here. Is `_hidden` referring strictly to the list of messages coming out of the analyzer (that appears above the footer)? If so, I'd rename this to something more specific.

Reply:
My thinking was that this scoped to AnalysisResultsController, so it represents whether or not the analysis results are hidden or shown, so `_analysisResultsHidden` might be redundant

Reply:
It's possible you have a different idea in your mind of what the class represents than I have in mine, and that's causing the confusion for me here.

When I look at this class I see an object that's playing the role of controller and wiring up multiple DOM elements (flash, message, toggle) so they work together. It doesn't represent a specific piece of the UI, but rather a connection between several pieces. From that perspective, the toggle, message, and results list are all equal partners, so when I get to lines 736-754 and see both "hide" and "hideToggle," I don't know at first glance what "hide" is intended to do.

This is a long-winded way of saying I'd recommend renaming `hide`/`show` to something more specific for benefit of future readers. 😄

Reply:
sounds good, changed to _flashHidden, showFlash, and hideFlash